src/main/java/org/dsa/iot/alarm/AlarmService.java

```diff
@@ -81,10 +81,14 @@ private void acknowledge(ActionResult event) {
             if (user == null) {
                 throw new NullPointerException("Missing User");
             }
-            UUID uuidObj = UUID.fromString(uuid.getString());
-            Alarming.getProvider().acknowledge(uuidObj, user.getString());
-            AlarmRecord rec = Alarming.getProvider().getAlarm(uuidObj);
-            rec.getAlarmClass().notifyAllUpdates(rec);
+            // 77513 - Acknowledge alarms with comma-separated UUIDs 
+            String items[] = uuid.getString().split(",");
+            for (String item : items) {
+                UUID uuidObj = UUID.fromString(item);
```

Comment:
I just did this small test:

  public static void main(String[] args) throws Exception
  {
      String[] test = ",abc,def,,ghi,".split(",");
      for (String s : test)
        System.out.println(">"+ s);
  }

This is this is what it printed:

>
>abc
>def
>
>ghi

So no trailing empty string, but a leading one.